### PR TITLE
Interactive feature

### PR DESCRIPTION
**Pull request by Rebelzob**
# To-do list Project

- Added a new js module called status.js to host status updates.
- Added event listener to the checkbox
- Added object update value
- Added Clear All function
- LocalStorage updated

### CODE REVIEW

File: src/index.html
Comment (Efiamotu-1) [inline src/index.html:1]: - [ ] In your readme file you have a `Project_presentation` section however there is no link to any presentation and this project milestone does not require a project presentation so I kindly suggest you remove this section from your readme file 👍 ✨

Comment (Efiamotu-1) [inline src/index.html:1]: - [ ] In your readme file, in the section `setup` you requested users to use `git clone`, but you didn't add your project link which they are to clone. I kindly suggest you make this important and necessary correction so your readme looks professional 👍 ✨

File: src/js/crud.js
Comment (Efiamotu-1) [inline src/js/crud.js:1]: - [ ] While you have imported new `JavaScript` files the name of the folder holding these files is called `js` rather it should be called `modules` seeing that you are modularizing your code by splitting & sharing the function into different files it is best to refer to the folder holding this files as `modules` 👍 ✨

File: src/js/status.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const updateCompletionStatus = (taskId, status, tasks) => {
+  tasks[taskId].completed = status;
+  return tasks;
+};
+
+export const clearAllCompleted = (tasks) => tasks.filter(task => !task.completed);
```
Comment (Efiamotu-1) [inline src/js/status.js:6]: - [ ] Currently after deleting all completed tasks the index of the remaining tasks left doesn't get reordered I kindly suggest you fix this.

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @Rebelzob , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Rebelzob, 
Approved 1

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes By Previous Reviewer :heavy_check_mark: 


## Optional suggestions
- [ ] Kindly consider fixing the broken links in the `table of contents` of your readme file to the `About Project` and `Author` sections as clicking on these links does not take the user to the desired sections.

The About Project Link
image

Authors Section Link 
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**